test/functionalTest/cases/3068_ngsi_v2_based_forwarding/check_no_overquerying.test

```diff
@@ -107,7 +107,7 @@ Date: REGEX(.*)
 
 03. Dump accumulator and check that E/A was forwarded (not including B or C)
 ============================================================================
-POST http://localhost:REGEX(\d+)/v2/op/query
+POST http://localhost:REGEX(\d+)/v2/op/query?limit=20&offset=0&options=count
```

Comment:
Existing tests should be modified by this PR.

If we change to rollback the change this way

```suggestion
POST http://localhost:REGEX(\d+)/v2/op/query
```

does the test pass?

Reply:
NTC (answered as general comment [here](https://github.com/telefonicaid/fiware-orion/pull/4275#issuecomment-1543420303))